packages/headless/src/features/commerce/recommendations/recommendations-state.ts

```diff
@@ -10,11 +10,18 @@ export interface RecommendationsState {
   products: Product[];
 }
 
-export const getRecommendationsInitialState = (): RecommendationsState => ({
-  slotId: '',
-  headline: '',
-  error: null,
-  isLoading: false,
-  responseId: '',
-  products: [],
-});
+/**
+ * An object in which each key is a slot identifier, and each value is the corresponding recommendations slice.
+ */
+export type RecommendationsState = Record<string, RecommendationsSlice>;
```

Comment:
Here, a `slotId` points to "recommendations", should it point to a "slot" instead?
```suggestion
export type RecommendationsState = Record<string, SlotSlice>;
```

Reply:
Good point! Applied this in https://github.com/coveo/ui-kit/pull/3801/commits/53b05cca263a9b71e330a2218bbe4e891d16023a